src/getHumanAge.test.js

```diff
@@ -3,8 +3,104 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  // below are postive tests for EC and BV test designs
+  // including different values=)
+  test('check if 0, 0 inputs give 0,0 output', () => {
+    const humanAges = getHumanAge(0, 0);
+
+    expect(humanAges)
+      .toEqual([0, 0]);
```

Comment:
You can use everywhere below:
```suggestion
    expect(getHumanAge(0, 0))
      .toEqual([0, 0]);
```

Reply:
Agree. Fixed.